revocation/revocation.go

```diff
@@ -100,6 +106,10 @@ type Options struct {
 	// OPTIONAL.
 	CRLHTTPClient *http.Client
 
+	// CRLCache is the cache client used to store the CRL. if not provided,
+	// no cache will be used.
+	CRLCache crlutil.Cache
```

Comment:
Since `CRLHTTPClient` is not in any release yet, should we merge `CRLHTTPClient` and `CRLCache` to `CRLFetcher`?

Reply:
Updated.